.github/workflows/export.yml

```diff
@@ -39,39 +38,74 @@ jobs:
         uses: actions/setup-python@v5.5.0
         with:
           # Version range or exact version of Python or PyPy to use, using SemVer's version range syntax. Reads from .python-version if unset.
+          # NOTE(pascalaubry): on Windows self-hosted runners, the first install fails:
+          # https://github.com/actions/setup-python/issues/969
```

Comment:
You might want to look at this, then: https://github.com/actions/setup-python/blob/55aad42e4674b58b2b2fb7d8e7552402d922b4e7/docs/advanced-usage.md#using-setup-python-with-a-self-hosted-runner